cmd/snap-seccomp/main.go

```diff
@@ -188,6 +188,23 @@ package main
 // #ifndef TIOCLINUX
 // #define TIOCLINUX 0x541C
 // #endif
+//
+//#include <linux/version.h>
+//#if LINUX_VERSION_CODE >= KERNEL_VERSION(3,19,0)
+// #include <linux/kcmp.h>
+// #if LINUX_VERSION_CODE < KERNEL_VERSION(4,13,0)
+//   #define KCMP_EPOLL_TFD 7
+// #endif
+//#else  // LINUX_VERSION_CODE >= KERNEL_VERSION(3,19,0)
+//  /* Define missing kcmp constants */
+//  #define KCMP_FILE 0
+//  #define KCMP_VM 1
+//  #define KCMP_FILES 2
+//  #define KCMP_FS 3
+//  #define KCMP_SIGHAND 4
+//  #define KCMP_IO 5
+//  #define KCMP_SYSVSEM 6
+//#endif // LINUX_VERSION_CODE >= KERNEL_VERSION(3,19,0)
```

Comment:
Looks like `KCMP_EPOLL_TFD` isn't being defined for kernel < 3.19.0 case. I think we should get the right result by getting rid of the nesting of the `#ifdef`s like so:

```suggestion
//#if LINUX_VERSION_CODE >= KERNEL_VERSION(3,19,0)
// #include <linux/kcmp.h>
//#else  // LINUX_VERSION_CODE >= KERNEL_VERSION(3,19,0)
//  /* Define missing kcmp constants */
//  #define KCMP_FILE 0
//  #define KCMP_VM 1
//  #define KCMP_FILES 2
//  #define KCMP_FS 3
//  #define KCMP_SIGHAND 4
//  #define KCMP_IO 5
//  #define KCMP_SYSVSEM 6
//#endif // LINUX_VERSION_CODE >= KERNEL_VERSION(3,19,0)
//#if LINUX_VERSION_CODE < KERNEL_VERSION(4,13,0)
//  #define KCMP_EPOLL_TFD 7
//#endif
```